### DISCUSSION TITLE
Using stored card with 3D secure does not trigger authentication

### DISCUSSION THREAD

**Discussion by VojtaSim** [swellstores/community#233] [Help] [answered]
When I use stored card for checkout that has 3D secure authentication enabled - more specifically in "always authenticate" mode (https://stripe.com/docs/testing#regulatory-cards) the order fails to submit.

I think it should handle payment action same as when the card is set up, e.g. with authorization dialog.

Do you have any idea how to workaround this problem?

**Comment by sags95** [comment, marked as answer]
@VojtaSim In these cases the customer would receive an email to confirm the payment through Stripe's 3ds flow. The order should not be blocked, but you should see this in the order activity. Are you seeing this on your end or are you getting an error when submitting the cart?

<img width="960" alt="Screenshot 2023-04-28 at 12 42 26 PM" src="https://user-images.githubusercontent.com/9522171/235205559-c13ee06b-08b7-4a43-b26c-b02481d1db5d.png">
